mlfinlab/tests/test_volatility_targeting.py

```diff
@@ -0,0 +1,42 @@
+"""
+Test volatility targeting method.
+"""
+
+import os
+import unittest
+
+import numpy as np
+import pandas as pd
+
+from mlfinlab.bet_sizing.targeting import volatility_targeting
+
+
+class TestVolatilityTargeting(unittest.TestCase):
+    """
+    Sets up the data to be used for the following tests.
+    """
+
+    def setUp(self):
+        project_path = os.path.dirname(__file__)
+
+        # Data set used for testing kelly formula.
+        stock_prices = pd.read_csv(project_path + '/test_data/stock_prices.csv', index_col=0)
+
+        spy_df = stock_prices['SPY']
```

Comment:
The result here is `pd.Series` and not the `np.ndarray` as the input type that `volatility_targeting` function expects